docs/apache-airflow/howto/operator/file.rst

```diff
@@ -22,7 +22,7 @@
 FileSensor
 ==========
 
-Use the :class:`~airflow.sensors.filesystem.FileSensor` to detect files appearing in your local
+Use the :class:`~airflow.providers.standard.sensors.filesystem.FileSensor` to detect files appearing in your local
```

Comment:
Please move this to standard provider docs. ref: https://github.com/apache/airflow/pull/43716

Reply:
Moved this to `docs/apache-airflow-providers-standard/operators/`.